### PR TITLE
Clarify reporting of documentation issues

### PR DESCRIPTION
**Pull request by ajdawson**
Addresses SciTools/scitools.org.uk/issues/159. Is this reasonable @jypeter?

### CODE REVIEW

**Comment by jypeter** [comment]
Shouldn't this go 2 lines below, to _Contribute to the documentation fixing typos_ ?

**Comment by ajdawson** [comment]
I wanted to catch this under reporting bugs in general, since documentation problems are also bugs, and I didn't want to duplicate. I also reasoned that mentioning it earlier in the listing would translate to more people seeing it...

**Comment by jypeter** [comment]
OK

File: docs/source/index.rst [hunk 1]
```diff
@@ -76,7 +76,7 @@ There are many ways to get involved in the development of cartopy:
 
  * If you write a paper which makes use of cartopy, please consider :doc:`citing <citation>` it.
  * If you publish the maps and plots, please consider the required :ref:`attribution of copyright  <referencing_copyright>` for the data. 
- * Report bugs to https://github.com/SciTools/cartopy/issues (please look to see if there are any outstanding
+ * Report bugs and problems with the code or documentation to https://github.com/SciTools/cartopy/issues (please look to see if there are any outstanding
```
Comment (dkillick) [inline docs/source/index.rst]: To me, "Report bugs and problems with the code" sounds a little tautological. Out of interest, what sort of things do you consider to be a _problem with the code_ that isn't also a _bug_?

Comment (ajdawson) [inline docs/source/index.rst]: Many of the tickets here are not bug reports, but other things like feature requests, clarifications, discussions about potential extensions and optimizations. That kind of thing.

**Comment by ajdawson** [comment]
You are probably right though, best be sure! How's that update @jypeter?

File: docs/source/index.rst (hunk 1 above)
Comment (jypeter) [inline docs/source/index.rst]: Well, I'm good as long as there is a link to a place where users can easily improve the documentation (if they feel like it)

**Comment by pelson** [comment]
Thanks @ajdawson. Sorry for the delay.

**Pull request merged**